### ISSUE TITLE
Multiple input tensors error

### ISSUE THREAD

**Issue by Satokaheni** [fchollet/keras#7251]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [x ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [ x] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [x ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [x ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by Satokaheni** [comment]
I'm trying to follow the multi-inputs guide on the website but keep getting the error that my two inputs aren't tensors. 
` 
init = Input(shape=(maxlen,))
        #embed the messages
        embed = Embedding(self.max_features, self.embedding_size, input_length=maxlen)(init)
        drop1 = Dropout(.25)(emebed)
        #create convolutional layers
        con = Conv1D(filters=self.nb_filters, kernel_size=self.filter_length, strides=2, padding='valid', activation='relu')(drop1)
        con = Conv1D(filters=self.nb_filters, kernel_size=self.filter_length, strides=2, padding='valid', activation='relu')(con)
        #pooling layer
        pool = MaxPooling1D(pool_size=self.pool_length)(con)
        #create LSTM layers
        bilstm = Bidirectional(LSTM(self.lstm_output_size))(pool)
        #Dense layers
        d = Dense(200, activation='relu')(bilstm)
        d = Dense(200, activation='relu')(d)
        d = Dense(200, activation='relu')(d)
        #second input
        auth_init = Input(shape=(1,))
        auth_dense = Dense(200, input_shape=(1,))(auth_init)
        concat = concatenate([init, auth_dense])
        #last dropout Layer
        drop2 =Dropout(.25)(concat)
        #output Layer
        output = Dense(3, activation='softmax')(drop2)
        model = Model(inputs=[init, auth_init], outputs=[output])
`
